docs-2.0/nebula-studio/deploy-connect/st-ug-deploy.md

```diff
@@ -128,13 +128,15 @@ Nebula Graph Studio（ 以下简称 Studio ）支持云端或本地部署。云
 2. 使用`sudo rpm -i <rpm>`命令安装RPM包。
 
    例如，安装Studio {{ studio.base220 }} 版本需要运行以下命令：
+
    ```bash
    $ sudo rpm -i nebula-graph-studio-2.2.0-1.x86_64.rpm
```

Comment:
220

Reply:
The variable name defined in yaml is base220.